.github/workflows/_reusable_workflows.yml

```diff
@@ -0,0 +1,12 @@
+# This is a dummy workflow that's just here to add a bit of organization to the workflow list.
+# Too bad they don't allow you to hide workflows or something.
+name: " 🚫 |----- Reusable Worfklows Below"
+on:
+  workflow_call:
+
+jobs:
+  nothing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
Is this sorta needlessly costly to run? I can't remember the syntax but iirc there's some output helpers on GitHub Actions, so maybe we could just print "ok" instead?

Reply:
I don't really expect anyone to ever actually run this on since it's just there to be a visible divider, but it's probably not a bad idea to make it do as little as possible, just in case.